tests/test_completion.py

```diff
@@ -209,6 +209,40 @@ def test_shell_command_completion(cmd2_app):
     begidx = endidx - len(text)
     assert cmd2_app.complete_shell(text, line, begidx, endidx) == expected
 
+def test_default_to_shell_command_completion(request):
```

Comment:
This test is failing for Python 2.7 on Linux and for all versions of Python on Windows.